arelle/ModelDocument.py

```diff
@@ -420,7 +420,7 @@ def create(modelXbrl, type, uri, schemaRefs=None, isEntry=False, initialXml=None
     elif type == Type.INSTANCE:
         # modelXbrl.uriDir = os.path.dirname(normalizedUri)
         Xml = ('<nsmap>{}'
-               '<xbrl xmlns="http://www.xbrl.org/2003/instance"'
+               '<xbrli:xbrl xmlns:xbrli="http://www.xbrl.org/2003/instance"'
```

Comment:
Not strictly needed, but for consistency it would be good to add the prefix to the closing element on line 429 below:
`Xml += '</xbrli:xbrl></nsmap>'`